python/dgl/graphbolt/minibatch.py

```diff
@@ -231,6 +231,20 @@ def blocks(self):
             self.sampled_subgraphs[0].sampled_csc, Dict
         )
 
+        # casts to minimum dtype in-place and returns self.
+        def cast_to_minimum_dtype(v: CSCFormatBase):
+            # Checks if number of vertices and edges fit into an int32.
+            if (
+                max(v.indptr.size(0) - 2, v.indices.size(0))
```

Comment:
`-1` or `-2`?

Reply:
-1 gives num_nodes and -2 gives max vertex id stores in the indices array. Thus, we need -2. Size of indices gives the max value stores in indptr array. So we are taking max of the max value stored in either array.